raster-foundry/raster-foundry
# Simplify adding scenes to projects UI

## Overview

This PR removes the modal that was previously used to add scenes to a project in favor of a more streamlined approach.

**Previous workflow:**
- select which scenes you want to add
- click the add scenes button
- confirm you want those scenes
- click the confirm button

**New workflow:**
- find the scene you want and click to add it to the project


### Checklist


### Demo

The new '+' button for each item:
![image](https://user-images.githubusercontent.com/2442245/44677372-c44ffb80-aa03-11e8-8e75-c2fe175fe216.png)


Indicating that scenes are already part of the project:
![image](https://user-images.githubusercontent.com/2442245/44677316-9ff41f00-aa03-11e8-9fe0-ebc66da209ce.png)

Messaging for non-RF repositories:
![image](https://user-images.githubusercontent.com/2442245/44677439-eba6c880-aa03-11e8-8220-289c385b6fec.png)

## Testing Instructions

 * Spin up the front-end
 * Add scenes to projects from various repos
 * Make sure import message is visible when browsing third-party repositories
 * Try adding all listed scenes
 * Confirm that this is great

Closes #3423 
Closes #3942 
Closes #3698
